Gopkg.toml

```diff
@@ -114,8 +114,9 @@
   name = "github.com/juju/romulus"
 
 [[constraint]]
-  revision = "6570bd8f8541d15ce654437084f8edcd7b76637b"
+  revision = "fe4dde6a8d8a5c36350355052d6e04c427c6efcf"
   name = "github.com/juju/testing"
+  source = "github.com/SimonRichardson/testing"
```

Comment:
If we need a testing patch, we probably just want to update juju/testing first?

Reply:
So I do this normally to get ok'd from reviewers that this is the right way to do this, so I don't have rev the testing lib and once I've got signed off with that, I land testing repo then update this repo.